hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/services/TestAbfsInputStream.java

```diff
@@ -192,6 +198,95 @@ public TestAbfsInputStream() throws Exception {
     ReadBufferManager.getBufferManager().setThresholdAgeMilliseconds(REDUCED_READ_BUFFER_AGE_THRESHOLD);
   }
 
+  private void writeBufferToNewFile(Path testFile, byte[] buffer) throws IOException {
+    AzureBlobFileSystem fs = getFileSystem();
+    fs.create(testFile);
+    FSDataOutputStream out = fs.append(testFile);
+    out.write(buffer);
+    out.close();
+  }
+
+  private void verifyOpenWithProvidedStatus(Path path, FileStatus fileStatus,
+      byte[] buf, AbfsRestOperationType source)
+      throws IOException, ExecutionException, InterruptedException {
+    byte[] readBuf = new byte[buf.length];
+    FSDataInputStream in = getFileSystem().openFileWithOptions(path,
```

Comment:
Use the required way of usage of this
FutureDataInputStreamBuilder builder = FileSystem#openFile(Path)
 Path p =new Path("....");
    FutureDataInputStreamBuilder builder = fs.openFile(p);
    builder.withFileStatus(status);
    FSDataInputStream fsDataInputStream = builder.build().get();

Reply:
done